_install-and-configure/configuring-opensearch/cluster-settings.md

```diff
@@ -116,6 +116,20 @@ OpenSearch supports the following cluster-level shard, block, and task settings:
 
 - `cluster.persistent_tasks.allocation.recheck_interval` (Time unit): The cluster manager automatically checks whether persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. Default is `30 seconds`, with a minimum of `10 seconds` being required. 
 
+## Cluster-level slow log settings
+
+For more information, see [Search request slow logs]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/logs/#search-request-slow-logs).
```

Comment:
Anyway to verify this URL is correct? It is contingent on the changes to `logs.md` in this same PR.

Reply:
The URL is correct. It's verified by the link checker CI and I checked locally by running the website on localhost.